src/content/contributing/translation-program/translatathon/index.md

```diff
@@ -0,0 +1,191 @@
+
+We will also be running anti-machine translation measures, with Crowdin providing some tools that automatically detect machine translations.
+
+While translation quality will not play a critical role in the scoring, any participants found using machine translation or suggesting low-quality and inaccurate translations will be disqualified and not eligible to compete for prizes!
+
+If you are participating in the Translatathon as part of a team, try to pick your teammates wisely. Any team members that are disqualified based on the above criteria will results in your team losing points, putting you at a disadvantage against other teams.
+
+The evaluation period will take one week after the translation period ends, and results & winners will be announced on August 31st.
+
+All translations will also be subject to a thorough review before being added to the website.
+
+## Terms & conditions {#Terms-and-Conditions}
+
+_The Translatathon is an experimental initiative by ethereum.org to incentivize and reward contributions to the ethereum.org Translation program. We reserve the right to modify the rules or cancel the event at any time, and the eligibility, score, and distribution of prizes are at the sole and final discretion of the ethereum.org team._
+
+- By submitting the application form, you consent to sharing all the mandatory information with the organizers. All of the information you provide will be kept private between yourself and the ethereum.org team, but your Crowdin username and profile image will be used in public announcements of the results and winners.
+
+- Using machine translation or suggesting incorrect and inaccurate translations will lead to disqualification and make you ineligible to compete for prizes. The ethereum.org team reserves the right to disqualify participants based on the above, and all such decisions are final.
+
+- Any suggested translations for strings that have already been translated or reviewed will not be counted towards your final score.
+
+- Prize winners will be subject to KYC before receiving prizes, and any taxes this might incur are the sole responsibility of the participant.
+
+- We are unable to issue prizes to participants from countries that are on sanctions lists (e.g. North Korea, Syria, Iran, and Cuba).
+
+## Frequently asked questions {#FAQ}
+
+<ExpandableCard title="Who can I contact if I need help or have questions, feedback, or ideas?" eventCategory="/translation-program/translatathon" eventName="clicked Who can I contact if I need help or have questions, feedback, or ideas?">
+
+- In Crowdin, you can send a direct message to [Ethereum.org Team](https://crowdin.com/profile/ethdotorg)
+- On the [ethereum.org Discord](https://www.ethereum.org/discord), you can send a message in the **#translatathon** & **#translate** channels
```

Comment:
..and similar to above, bold items would be switched to `<strong>Text here</strong>`, and italics would be `<em>Text here</em>` as needed